x-pack/plugins/ml/public/application/explorer/anomaly_timeline.tsx

```diff
@@ -310,63 +287,79 @@ export const AnomalyTimeline: FC<AnomalyTimelineProps> = React.memo(
             onMouseLeave={onSwimlaneLeaveHandler}
             data-test-subj="mlAnomalyExplorerSwimlaneOverall"
           >
-            {showOverallSwimlane && (
-              <SwimlaneContainer
-                filterActive={filterActive}
-                maskAll={maskAll}
-                timeBuckets={timeBuckets}
-                swimlaneCellClick={swimlaneCellClick}
-                swimlaneData={overallSwimlaneData as OverallSwimlaneData}
-                swimlaneType={'overall'}
-                selection={selectedCells}
-                swimlaneRenderDoneListener={swimlaneRenderDoneListener}
-                onResize={(width) => explorerService.setSwimlaneContainerWidth(width)}
-              />
-            )}
+            <SwimlaneContainer
+              filterActive={filterActive}
+              maskAll={maskAll}
+              timeBuckets={timeBuckets}
+              swimlaneCellClick={swimlaneCellClick}
+              swimlaneData={overallSwimlaneData as OverallSwimlaneData}
+              swimlaneType={'overall'}
+              selection={selectedCells}
+              swimlaneRenderDoneListener={swimlaneRenderDoneListener}
+              onResize={(width) => explorerService.setSwimlaneContainerWidth(width)}
+              isLoading={loading}
+              noDataWarning={<NoOverallData />}
+            />
           </div>
 
+          <EuiSpacer size="m" />
+
           {viewBySwimlaneOptions.length > 0 && (
             <>
-              {showViewBySwimlane && (
-                <>
-                  <EuiSpacer size="m" />
-                  <div
-                    className="ml-explorer-swimlane euiText"
-                    onMouseEnter={onSwimlaneEnterHandler}
-                    onMouseLeave={onSwimlaneLeaveHandler}
-                    data-test-subj="mlAnomalyExplorerSwimlaneViewBy"
-                  >
-                    <SwimlaneContainer
-                      filterActive={filterActive}
-                      maskAll={
-                        maskAll &&
-                        !hasMatchingPoints({
-                          filteredFields,
-                          swimlaneData: viewBySwimlaneData,
-                        })
+              <>
+                <div
+                  className="ml-explorer-swimlane euiText"
```

Comment:
Classname should be `mlExplorerSwimlane` to match our EUI standards

Reply:
I haven't introduced this class in this PR, but renamed in https://github.com/elastic/kibana/pull/70063/commits